### PR TITLE
Gradebook api calls

### PR DESCRIPTION
**Pull request by Goff-Davis**

### CODE REVIEW

**Review by weining-li — DISMISSED** [review]
i can't see the gradebook so i'll assume everything else is fine :cactus:

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -22,36 +22,39 @@ function App(): ReactElement {
...
-
-    //when established, join the room as the professor
-    websocket.onopen = () => {
-      websocket.send(
-        JSON.stringify({
-          action: "professorJoinRoom",
-          courseId: state.courseId,
-        })
+    if (state.courseId) {
+      console.log(state.courseId);
```
Comment (ljoks) [inline frontend/src/App.tsx]: remove console log

Comment (Goff-Davis) [reply frontend/src/App.tsx]: You're looking at an old commit, that's already removed

**Review by weining-li — DISMISSED** [review]
¯\_(ツ)_/¯

**Review by kennygperez — APPROVED** [review]

**Review by kennygperez — APPROVED** [review]

**Pull request merged**